runtime/queries/apex/locals.scm

```diff
@@ -22,7 +22,8 @@
 ; loops
 
 (for_statement) @local.scope
-(for_statement         ; "for" body in case there are no braces
+; "for" body in case there are no braces
+(for_statement  
```

Comment:
sorry, missed that: can you drop that change?

Reply:
Sorry. Fixed that again today